index.js

```diff
@@ -14,14 +14,71 @@ mongoose
     useNewUrlParser: true,
     useUnifiedTopology: true
   })
+
   .then(self => {
     console.log(`Connected to the database: "${self.connection.name}"`);
     // Before adding any documents to the database, let's delete all previous entries
     return self.connection.dropDatabase();
   })
+
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
+    Recipe.create({
+        title: 'recipe1',
+        cuisine: "Asian"
+      })
+      .then((response) => {
+        console.log(response)
+      })
+      .catch((err) => {
+        console.log('oh no:', err)
+      })
+
+    Recipe.insertMany(data)
+      .then((response) => {
+        response.forEach(recipe => {
+          console.log(recipe.title)
+        })
+
+        let updateRigatoni = Recipe.findOneAndUpdate({
+          title: 'Rigatoni alla Genovese'
+        }, {
+          $set: {
```

Comment:
findOneAndUpdate knows you will be updating a property in the second argument. So you don't need to specify this with the $set.